src/shared/components/Tooltip/Tooltip.tsx

```diff
@@ -30,7 +30,7 @@ export const Tooltip: React.FC<TooltipProps> = ({
   arrowDisabled,
   className,
 }) => {
-  const [isVisible, setIsVisible] = useState(false)
+  const [isVisible, setIsVisible] = useState(true)
```

Comment:
Where is this change coming from?

Reply:
damn it 😅 . I should more carefully check what I'm committing.